docs/install/knative-with-any-k8s.md

```diff
@@ -496,6 +496,12 @@ You can find a number of sample on how to get started with Knative Eventing [her
 
 Knative provides a bundle of monitoring components that can be used to make the Serving and Eventing components more observable.
 
+Before you install any specific monitoring components, you must first install the core monitoring pieces:
+
+```bash
+kubectl apply --filename https://github.com/knative/serving/releases/download/{{< version >}}/monitoring-core.yaml
```

Comment:
```suggestion
kubectl apply --filename {{< artifact repo="serving" file="monitoring-core.yaml" >}}
```

This is something that landed in parallel to your change, which lets us use nightly builds on the pre-release page.

Reply:
done!